wordpress-mobile/WordPress-iOS
# Reader: Add Reader P2 deep link.

Ref: #15343 

This adds a deep link for the Reader P2 section.

To test:
- Run the app in a simulator.
- Log in with an account that is following a non-A8C P2.
- In the command line, enter `xcrun simctl openurl booted https://wordpress.com/read/p2`.
- Back in the simulator, verify the app switched to the Reader > P2 view. Rejoice.

PR submission checklist:
